src/content/changes/highlights/2024-07-21.md

```diff
@@ -0,0 +1,186 @@
+        - ReclaimPolicy added in the Bucket CR to manage the data clean up (retain or delete).
+        - Add a finalizer on the Azure secret to prevent its deletion.
+        - Empty all the objects in the S3 bucket in case of bucket deletion.
+- [observability-bundle](https://github.com/giantswarm/observability-bundle)
+    - [1.5.1](https://github.com/giantswarm/observability-
+bundle/compare/v1.5.0...v1.5.1)
+        - Upgrade `prometheus-operator-crd` to 11.0.1.
+    - [1.5.0](https://github.com/giantswarm/observability-
+bundle/compare/v1.4.0...v1.5.0)
+        - Add `alloy` v0.3.0 as `alloy-logs`
+        - prometheus-operator will not check promql syntax for prometheusRules that are labelled `application.giantswarm.io/prometheus-rule-kind: loki`
+- [observability-operator](https://github.com/giantswarm/observability-operator)
+    - [0.3.0](https://github.com/giantswarm/observability-
+operator/compare/v0.2.0...v0.3.0)
+        - Delete monitoring resources if monitoring is disabled at the installation or cluster level using the giantswarm.io/monitoring label.
+- [prometheus-operator-crd](https://github.com/giantswarm/prometheus-operator-crd)
+    - [11.0.1](https://github.com/giantswarm/prometheus-operator-
+crd/compare/v11.0.0...v11.0.1)
+        - Add `helm.sh/resource-policy: keep` annotation to all CRDs to avoid deletion during Helm operations.
+- [prometheus-rules](https://github.com/giantswarm/prometheus-rules)
+    - [4.8.0](https://github.com/giantswarm/prometheus-
+rules/compare/v4.7.0...v4.8.0)
+        - Move alloy to monitoring namespace
+    - [4.7.0](https://github.com/giantswarm/prometheus-
+rules/compare/v4.6.3...v4.7.0)
+        - Support for loki rules to management clusters in alloy config
+        - grafana datasource for MC loki ruler
+        - Make dns-operator-azure capz only.
+        - Fix PromtailDown alert to fire only when the node is ready.
+- [kube-downscaler-app](https://github.com/giantswarm/kube-downscaler-app)
+    - [0.3.0](https://github.com/giantswarm/kube-downscaler-
+app/compare/v0.2.0...v0.3.0)
+        - Push `kube-downscaler` app to all collections.
+    - [0.2.0](https://github.com/giantswarm/kube-downscaler-
+app/compare/v0.1.0...v0.2.0)
+        - Added `enabled` field in values to disable whole chart if needed.
+
+
+- [dex-app](https://github.com/giantswarm/dex-app)
+    - [1.42.11](https://github.com/giantswarm/dex-app/compare/v1.42.10...v1.42.11)
+        - Default `ingress.tls.clusterIssuer` values to `letsencrypt-giantswarm`
+        - Update `cert-manager.io/cluster-issuer` annotation to use default.
+- [teleport-kube-agent-app](https://github.com/giantswarm/teleport-kube-agent-app)
+    - [0.9.2](https://github.com/giantswarm/teleport-kube-agent-
+app/compare/v0.9.1...v0.9.2)
+        - Introduced `podAntiAffinity` so `teleport-kube-agent` pods run on different `control-plane` nodes also increased the number of replicas to 3 to maintain better high availability.
```

Comment:
@gawertm  These are from Bigmac. Please adapt the content to the standard format (name, version, description)
[example](https://raw.githubusercontent.com/giantswarm/docs/main/src/content/changes/highlights/2024-02-15.md)